src/displayapp/screens/Dice.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "displayapp/apps/Apps.h"
+#include "displayapp/screens/Screen.h"
+#include "displayapp/widgets/Counter.h"
+#include "displayapp/Controllers.h"
+#include "Symbols.h"
+
+#include <array>
+#include <random>
+
+#define ROLL_HYSTERESIS 10
```

Comment:
We try to avoid using the preprocessor as much as possible, so I'd replace this with a static constexpr value in the class.

Reply:
Thanks! Great point. Corrected in https://github.com/InfiniTimeOrg/InfiniTime/pull/1326/commits/d4c69757e4f8be9e3b5059cd38f2f8a90a4ba5fa